cmd/git-sync/main.go

```diff
@@ -181,14 +211,30 @@ func main() {
 		}
 	}
 
+	if *flHTTPEnable {
+		ln, err := net.Listen("tcp", *flHTTPBind)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "ERROR: unable to bind HTTP endpoint: %v\n", err)
+		}
+		go func() {
+			if *flHTTPMetrics {
+				http.Handle("/metrics", promhttp.Handler())
+			}
+			http.Serve(ln, http.DefaultServeMux)
```

Comment:
Should we have a do-nothing success handler for / ?

Reply:
From a prometheus perspective most implementations I've seen default to 404 as there is nothing serving so why give back a 200 return.

Reply:
I'm fine to add a dumb liveliness handler to `/` but that really isn't part of this metrics PR IMO -- but I'd be more than happy to add it in.

Reply:
dumb liveliness handler added

Reply:
As per other comment, the ideal review sequence would be to add HTTP with a do-nothing / handler in one PR, followed by adding metrics in a different PR.  I'll take them together, if need be.